.github/fabricbot.json

```diff
@@ -1,3600 +0,0 @@
-        "packageRegex": "(v.*)",
-        "packageVersionGroup": 1
-      },
-      "disabled": false
-    },
-    {
-      "taskType": "trigger",
-      "capabilityId": "IssueResponder",
-      "subCapability": "IssuesOnlyResponder",
-      "version": "1.0",
-      "id": "rk_usYpO0",
-      "config": {
-        "conditions": {
-          "operator": "and",
-          "operands": [
-            {
-              "operator": "or",
-              "operands": [
-                {
-                  "name": "hasLabel",
-                  "parameters": {
-                    "label": "bug :bug:"
-                  }
-                },
-                {
-                  "name": "hasLabel",
-                  "parameters": {
-                    "label": "question :grey_question:"
-                  }
-                },
-                {
-                  "name": "hasLabel",
-                  "parameters": {
-                    "label": "feature request :mailbox_with_mail:"
-                  }
-                }
-              ]
-            },
-            {
-              "operator": "or",
-              "operands": [
-                {
-                  "name": "isActivitySender",
-                  "parameters": {
-                    "user": "Kyaa-dost"
-                  }
-                },
-                {
-                  "name": "isActivitySender",
-                  "parameters": {
-                    "user": "michael-hawker"
-                  }
-                },
-                {
-                  "name": "isActivitySender",
-                  "parameters": {
-                    "user": "azchohfi"
-                  }
-                },
-                {
-                  "name": "isActivitySender",
-                  "parameters": {
-                    "user": "RosarioPulella"
-                  }
-                }
-              ]
-            },
-            {
-              "operator": "and",
-              "operands": [
-                {
-                  "name": "isAction",
-                  "parameters": {
-                    "action": "reopened"
-                  }
-                },
-                {
-                  "operator": "not",
-                  "operands": [
-                    {
-                      "name": "isPartOfProject",
-                      "parameters": {}
-                    }
-                  ]
-                },
-                {
-                  "operator": "not",
-                  "operands": [
-                    {
-                      "name": "isAssignedToSomeone",
-                      "parameters": {}
-                    }
-                  ]
-                }
-              ]
-            }
-          ]
-        },
-        "eventType": "issue",
-        "eventNames": [
-          "issues",
-          "project_card"
-        ],
-        "taskName": "Add Need Triage label when opening issue by certain activity sender",
```

Comment:
This task didn't get ported over, presumably because it references `isPartOfProject` which is not supported.